aip/general/0193.md

```diff
@@ -69,13 +69,27 @@ and the method **should** put partial failure information in the metadata
 message. The errors themselves **must** still be represented with a
 [`google.rpc.Status`][Status] object.
 
+### Permission Denied
+
+If the API requires authentication and authorization, the API **must** first
+check whether the user has the permission to perform the operation, and **must**
+error with `PERMISSION_DENIED` (HTTP 403) if so.
+
+Checking for the permissions and verifying first ensures that the caller cannot
```

Comment:
I think this can be much simpler.

"Service should return `NOT_FOUND` (HTTP 404) after successful access verification to prevent unauthorized actors from inferring existence of a resource."

Reply:
Same: IMHO, we should keep the existing text.